snapcraft/tests/test_lifecycle.py

```diff
@@ -678,6 +686,36 @@ def test_prime_with_build_info_records_snapcraft_yaml(self):
 description: test
 confinement: strict
 grade: stable
+
+
```

Comment:
Any idea where that extra space is coming from? I was going to suggest fixing it before I realized it's the expected output...